### PR TITLE
[Feature] Add readme for Improved-DDPM

### PR DESCRIPTION
**Pull request by LeoXing1996**

### CODE REVIEW

File: configs/improved_ddpm/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+| Improve-DDPM 64x64 Dropout=0.3 | ImageNet1k | 13.4094 | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/improve_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k.py) | [model](https://download.openmmlab.com/mmgen/improved_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k_20220103_224427-7bb55975.pth)&#124; [log](https://download.openmmlab.com/mmgen/improved_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k_20220103_224427-7bb55975.json) |
+
+`FID` comparison with official:
+
+| Dataset  | CIFAR10 | ImageNet1k-64x64 |
+|:--------:|:-------:|:----------------:|
+|   Ours   | 3.8848  |     13.5181      |
+| Official |  3.19   |       19.2       |
+
+For FID evaluation, differences between PyTorch Studio GAN and ours are mainly on the selection of real samples. In MMGen, we follow the pipeline of BigGAN, where the whole training set is adopted to extract inception statistics. Besides, we also use [Tero's Inception](https://nvlabs-fi-cdn.nvidia.com/stylegan2-ada-pytorch/pretrained/metrics/inception-2015-12-05.pt) for feature extraction.
```
Comment (plyfager) [inline configs/improved_ddpm/README.md:50]: So StudioGAN uses 50k real samples while MMGEN and BigGAN use whole training set? If true, maybe you should specify it.

Comment (plyfager) [reply configs/improved_ddpm/README.md:50]: Also adding links for `StudioGAN` and `BigGAN` seems better.

File: configs/improved_ddpm/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+You can download the preprocessed inception state by the following url: [CIFAR10](https://download.openmmlab.com/mmgen/evaluation/fid_inception_pkl/cifar10.pkl) and [ImageNet1k-64x4](https://download.openmmlab.com/mmgen/evaluation/fid_inception_pkl/imagenet_64x64.pkl).
```
Comment (plyfager) [inline configs/improved_ddpm/README.md:52]: TYPO: `ImageNet1k-64x4`

File: configs/improved_ddpm/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+| Improve-DDPM 64x64 Dropout=0.3 | ImageNet1k | 13.4094 | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/improve_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k.py) | [model](https://download.openmmlab.com/mmgen/improved_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k_20220103_224427-7bb55975.pth)&#124; [log](https://download.openmmlab.com/mmgen/improved_ddpm/ddpm_cosine_hybird_timestep-4k_drop0.3_imagenet1k_64x64_b8x16_1500k_20220103_224427-7bb55975.json) |
+
+`FID` comparison with official:
+
+| Dataset  | CIFAR10 | ImageNet1k-64x64 |
+|:--------:|:-------:|:----------------:|
+|   Ours   | 3.8848  |     13.5181      |
```
Comment (plyfager) [inline configs/improved_ddpm/README.md:47]: Mark better results in bold?

**Review by plyfager — CHANGES_REQUESTED** [review]
Good Job!

File: configs/improved_ddpm/README.md (hunk 1 above)
Comment (LeoXing1996) [reply configs/improved_ddpm/README.md:50]: Newest StudioGAN uses samples from the entire dataset without shuffle. I will update this to our readme related to FID.

**Pull request merged**